user_sync/connector/directory_ldap.py

```diff
@@ -310,7 +312,7 @@ def iter_users(self, base_dn, users_filter, extended_attributes):
         user_attribute_names.extend(self.user_email_formatter.get_attribute_names())
         user_attribute_names.extend(self.user_username_formatter.get_attribute_names())
         user_attribute_names.extend(self.user_domain_formatter.get_attribute_names())
-        user_attribute_names.append(six.text_type('memberOf'))
+        user_attribute_names.extend(self.user_memberof_format_formatter.get_attribute_names())
```

Comment:
We just need to append the value of our new config option here since we aren't doing formatting.